src/main/java/dev/dokan/dokan_java/conv/MaskValueEnum.java

```diff
@@ -0,0 +1,21 @@
+package dev.dokan.dokan_java.conv;
+
+
+public interface MaskValueEnum extends EnumInteger {
+
+    static <T extends Enum<T> & MaskValueEnum> T enumFromInt(final int value, final T[] enumValues) {
```

Comment:
the naming of the functions needs to be discussed (see `MountOptions.java`)

Reply:
I presume You meant to write `MountOption.java`.

Also what would be your suggestion for `enumFromInt`?

Reply:
Resolved with commit e042b3d44d5b944f198f2cd3a2dca89419f56e8f / 3114b10f633ea2c173307afff6119c208ba7d791